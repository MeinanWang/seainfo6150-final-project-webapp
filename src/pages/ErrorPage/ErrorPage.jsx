import React from "react";
import errorImage from "../../images/error.jpg";
import styles from './ErrorPage.module.css';

const ErrorPage = () => {
    return (
        <div className={styles.container}>
            <div className={styles.text}>
                <h1>Oops Sorry!</h1>
                <h2>404 Page not found</h2>
                <p>It seems that something went wrong...</p>
            </div>

            <img className={styles.image} src={errorImage} alt="error" />

        </div>
    )
}

export default ErrorPage;