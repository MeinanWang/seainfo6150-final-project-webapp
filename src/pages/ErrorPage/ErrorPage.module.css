.container{
    margin: 0 10% ;
}

.text{
    text-align: center;
}

.image{
    width: 100%;
}