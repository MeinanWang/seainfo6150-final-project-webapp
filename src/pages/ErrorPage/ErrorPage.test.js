import React from "react";
import { render } from "@testing-library/react";
import ErrorPage from "./ErrorPage.jsx";


it("renders the Error page correctly", () => {
    const { container } = render(<ErrorPage />);
    expect(container).toMatchSnapshot();
});