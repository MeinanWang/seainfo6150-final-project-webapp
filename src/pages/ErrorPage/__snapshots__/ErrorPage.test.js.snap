// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Error page correctly 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="text"
    >
      <h1>
        Oops Sorry!
      </h1>
      <h2>
        404 Page not found
      </h2>
      <p>
        It seems that something went wrong...
      </p>
    </div>
    <img
      alt="error"
      class="image"
      src="error.jpg"
    />
  </div>
</div>
`;
